_docker/drupal/drupal-filesystem/web/themes/custom/rhdp/templates/system/html.html.twig

```diff
@@ -247,8 +247,15 @@ window.NREUM||(NREUM={}),__nr_require=function(t,n,e){function r(e){if(!n[e]){va
 
     <css-placeholder token="{{ placeholder_token }}">
     {% if (rhd_sentry_track) %}
-    <script src="{{ rhd_sentry_script }}" crossorigin="anonymous"></script>
-    <script>Raven.config('{{ rhd_sentry_code }}', { environment: '{{rhd_environment}}', whitelistUrls: [/redhat\.com/] }).install()</script>
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=Promise%2CObject.assign%2CString.prototype.includes%2CNumber.isNaN"></script>
+    <script src="https://browser.sentry-cdn.com/5.7.1/bundle.min.js" integrity="sha384-pHnCJ3bbzJ+LzI19wAdyBvGAjJ5vNAC59uxeRpw5WpgkSlZUQstEADxkSbBnI95q" crossorigin="anonymous"></script>
+    <script>
+    Sentry.init({ 
+      dsn: '{{ rhd_sentry_code }}',
+      environment: '{{rhd_environment}}',
+      whitelistUrls: [ /developers\.redhat\.com/ ]
+    });
+    </script>
```

Comment:
I've created a ticket here: https://github.com/redhat-developer/developers.redhat.com/issues/3307 to move all of these scripts that are defined in the `html.html.twig` files into Drupal libraries. I think that would make it clearer, with respect to code organization, and hopefully we can optimize the way that we contextually load some of these scripts and styles down the road.

Reply:
That is a net-new feature in my opinion though, and I am happy with these changes. It does not particularly matter to whether the `rhd_sentry_script` variable is used or the URL is manually declared here. It looks like the variable isn't even set by the build processes or dynamic, based upon the environment, so I do not see much of a difference personally.

Reply:
@jordanpagewhite The value of the `rhd_sentry_script` variable is set in `rhdp.theme` and `rhdp2.theme` and is read from `rhd.settings.php`

- https://github.com/redhat-developer/developers.redhat.com/blob/master/_docker/drupal/drupal-filesystem/web/themes/custom/rhdp/rhdp.theme#L154-L156

and

- https://github.com/redhat-developer/developers.redhat.com/blob/master/_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhdp2.theme#L742-L744

So if we want to hard-code it into the template (which is fine if it never needs to vary between environments) then I'd like to see us clear up the properties from `rhd.settings.php`